thesis-de/content-de.tex

```diff
@@ -110,6 +110,14 @@ \subsection{Datenerfassungssysteme}
 \subsection{Datenbanksysteme}
 
 \subsection{Frontends}
+Die Weboberfläche muss ausgewählten Benutzern eine Übersicht über die Nutzung einzelner Virtuellen oder Physische Maschinen bereitstellen.
```

Comment:
please don't use more than 80 chars in a row, vim can highlight you these violations by setting:
`:set cc=80`